src/ucs/sys/topo/base/topo.c

```diff
@@ -308,9 +308,6 @@ static void ucs_topo_sys_root_distance(ucs_sys_dev_distance_t *distance)
 {
     distance->latency = 500e-9;
     switch (ucs_arch_get_cpu_model()) {
-    case UCS_CPU_MODEL_INTEL_ICELAKE:
```

Comment:
i would revert only this part, and keep ICELake CPU detection

Reply:
Done.